src/components/cutscenes/PokeCenterCutscene.ts

```diff
@@ -177,6 +177,18 @@ export class PokeCenterCutscene<TGameStartr extends FullScreenPokemon> extends C
                 this.gameStarter.scenePlayer.stopCutscene();
             });
         this.gameStarter.menuGrapher.setActiveMenu("GeneralText");
+        const mapLocation = this.gameStarter.itemsHolder.getItem("map");
+        if (mapLocation === "Pallet Town") {
```

Comment:
I don't like how this is hardcoding it to only allow `Pallet Town` to have this behavior. The `PokeCenterCutscene` file shouldn't be the place that decides whether an area is special; the area should.